app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      before_action :authenticate_api_v1_user!
+      def index
+        @rents = Rent.all
+        render json: @rents
+      end
+
+      def create
+        @rents = Rent.new(params[:rent])
```

Comment:
`render json: Rent.create!(rent_params)`

1) define a rent_params using require rent and permitted params
2) rescue the invalid record exception in the application controller (just like the not found exception)